improver/ensemble_copula_coupling/constants.py

```diff
@@ -48,39 +48,48 @@
 # Calibrated ensemble reliability whilst preserving spatial structure.
 # Tellus Series A, Dynamic Meteorology and Oceanography, 66, 22662.
 
+# Grouped by parameter, then sorted alphabetically.
 BOUNDS_FOR_ECDF = {
-    "air_temperature": (Bounds((-100 - ABSOLUTE_ZERO, 60 - ABSOLUTE_ZERO), "Kelvin")),
-    "feels_like_temperature": (
-        Bounds((-100 - ABSOLUTE_ZERO, 60 - ABSOLUTE_ZERO), "Kelvin")
-    ),
-    "wind_speed": Bounds((0, 50), "m s^-1"),
-    "wind_speed_of_gust": Bounds((0, 200), "m s^-1"),
-    "air_pressure_at_sea_level": Bounds((86000, 108000), "Pa"),
-    "cloud_base_altitude_assuming_only_consider_cloud_area_fraction_greater_than_2p5_oktas": Bounds(
-        (-300, 20000), "m"
-    ),
+    # Cloud
     "cloud_area_fraction": Bounds((0, 1.0), "1"),
     "cloud_area_fraction_assuming_only_consider_surface_to_1000_feet_asl": Bounds(
         (0, 1.0), "1"
     ),
+    "cloud_base_altitude_assuming_only_consider_cloud_area_fraction_greater_than_2p5_oktas": Bounds(
+        (-300, 20000), "m"
+    ),
     "low_type_cloud_area_fraction": Bounds((0, 1.0), "1"),
-    "lwe_precipitation_rate": Bounds((0, 128.0), "mm h-1"),
-    "lwe_precipitation_rate_in_vicinity": Bounds((0, 128.0), "mm h-1"),
-    "rainfall_rate": Bounds((0, 128.0), "mm h-1"),
-    "rainfall_rate_in_vicinity": Bounds((0, 128.0), "mm h-1"),
-    "relative_humidity": Bounds((0, 1.2), "1"),
+    # Precipitation amount
     "lwe_thickness_of_precipitation_amount": Bounds((0, 0.5), "m"),
     "lwe_thickness_of_precipitation_amount_in_vicinity": Bounds((0, 0.5), "m"),
+    "lwe_thickness_of_sleetfall_amount": Bounds((0, 0.5), "m"),
     "lwe_thickness_of_snowfall_amount": Bounds((0, 0.5), "m"),
+    "thickness_of_rainfall_amount": Bounds((0, 0.5), "m"),
+    "air_pressure_at_sea_level": Bounds((86000, 108000), "Pa"),
```

Comment:
Should `air_pressure_at_sea_level` be moved down into the "Others" group?

Reply:
Definitely! Well done for spotting my deliberate mistake, to put a positive spin on it.